src/svg.rs

```diff
@@ -3,6 +3,12 @@ use svgdom::*;
 
 use crate::errors::*;
 
+pub enum SvgRectIdAttributes {
+    Id = 0,
```

Comment:
`Id = 0` yields to compilation warning about a dead code: `variant is never constructed: 'Id'`

Reply:
I've finally found some use for it in last commits)